atomics/T1552.001/T1552.001.yaml

```diff
@@ -189,3 +188,48 @@ atomic_tests:
       
     name: command_prompt
     elevation_required: true
+- name: Find Azure credentials
```

Comment:
Can we update these atomic names to some like : Discover Azure credentials using Find